examples/talker_listener_sandbox_docker.launch.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
```

Comment:
Should this file be in a separate commit? Not sure how it's specific to logging.

Reply:
I could move it to a separate commit. I used it to test logging

Reply:
Ok, that makes sense.